mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewId.kt

```diff
@@ -0,0 +1,38 @@
+package com.airbnb.mvrx
+
+import android.os.Bundle
+import java.util.*
+import kotlin.properties.ReadOnlyProperty
+import kotlin.reflect.KProperty
+
+class MvRxViewId : ReadOnlyProperty<MvRxView, String> {
+
+    private var value: String? = null
+
+    override fun getValue(thisRef: MvRxView, property: KProperty<*>): String {
+        val currentValue = value
```

Comment:
I know you all don't like using Kotlin scoping functions that much but this can be actually simplified like below. I was gonna send that but then decided to ask first. Let me know what you think

```
return value ?: generateUniqueId(thisRef).also { value = it }
```

Reply:
sure, I like that one liner :)

Reply:
Applied.